docs/topics/OAuth2.md

```diff
@@ -46,13 +46,13 @@ These are a list of all the OAuth2 scopes that Discord supports. Scopes that are
 | applications.commands.update | allows your app to update [Slash Commands](#DOCS_INTERACTIONS_SLASH_COMMANDS/) via this bearer token                                                  |
 
 > info
-> `guilds.join` and `bot` require you to have a bot account linked to your application. Also, in order to add a user to a guild, your bot has to already belong to that guild.
+> `guilds.join` and `bot` require you to have a bot account linked to your application. Also, to add a user to a guild, your bot must already belong to that guild.
 
 ## State and Security
 
-Before we dive into the semantics of the different OAuth2 grants, we should stop and discuss security, specifically the use of the `state` parameter. [Cross Site Request Forgery](https://en.wikipedia.org/wiki/Cross-site_request_forgery), or CSRF, and [Clickjacking](https://en.wikipedia.org/wiki/Clickjacking) are security vulnerabilities that must be addressed by individuals implementing OAuth. This is typically accomplished using the `state` parameter. `state` is sent in the authorization request and returned back in the response and should be a value that binds the user's request to their authenticated state. For example, `state` could be a hash of the user's session cookie, or some other nonce that can be linked to the user's session.
+Before we dive into the semantics of the different OAuth2 grants, we should stop and discuss security, especially the use of the `state` parameter. [Cross Site Request Forgery](https://en.wikipedia.org/wiki/Cross-site_request_forgery), or CSRF, and [Clickjacking](https://en.wikipedia.org/wiki/Clickjacking) are security vulnerabilities that must be addressed by individuals implementing OAuth. This is typically accomplished using the `state` parameter. `state` is sent in the authorization request and returned back in the response and should be a value that binds the user's request to their authenticated state. For example, `state` could be a hash of the user's session cookie or some other nonce that can be linked to the user's session.
```

Comment:
I'm not big on grammar, but "especially" gives me the impression that we'll be talking about more than what we're "especially" talking about. I think "specifically" actually fits better here.